Description: Remove value assignments to environment variable LD_RUN_PATH
 Value assignment to environment variable LD_RUN_PATH leads to setting RPATH in
 compiled binary/shared library.
 .
 And then Lintian reports about error (E: binary-or-shlib-defines-rpath).
Author: Janos Guljas <janos@debian.org>
Last-Update: 2013-02-07

--- a/plugins/python/uwsgiplugin.py
+++ b/plugins/python/uwsgiplugin.py
@@ -20,10 +20,8 @@ if not 'UWSGI_PYTHON_NOLIB' in os.enviro
     else:
         try:
             LDFLAGS.append("-L%s" % sysconfig.get_config_var('LIBDIR'))
-            os.environ['LD_RUN_PATH'] = "%s" % (sysconfig.get_config_var('LIBDIR'))
         except:
             LDFLAGS.append("-L%s/lib" % sysconfig.PREFIX)
-            os.environ['LD_RUN_PATH'] = "%s/lib" % sysconfig.PREFIX
 
 
     version = sysconfig.get_config_var('VERSION')
